src/utility/Ex3.py

```diff
@@ -0,0 +1,195 @@
+import json
+import random
+import numpy as np
+
+
+class Pedestrian:
+    def __init__(self, age, personality):
+        self.age = age
+        self.personality = personality
+        self.emotions = {
+            "pleasure": 0.75,
+            "surprise": 0.5,
+            "anger": -0.2,
+            "fear": -0.2,
+            "hate": -0.4,
+            "sad": -0.4
+        }
+
+
+class Patient(Pedestrian):
+    def __init__(self, age, personality, status, journey):
+        super().__init__(age, personality)
+        self.type = "Patient"
+        self.status = status
+        self.journey = journey
+
+    def toJSON(self):
```

Comment:
@@ I think you should follow: [Object to JSON Python](https://www.geeksforgeeks.org/convert-class-object-to-json-in-python/) and try it

Reply:
Now that you mentioned it, I'm afraid it might conflict with some custom data type from numpy library (Array instead of List, etc...).
But for now, I agree with him since we're only using primitive data type here, so please consider following it :D